input_formats: ["FP", "LLH"]
input_type: tcp
input_port: "21000"
tcp_ip: "10.0.2.1"
rate: 200
